apocalypse_sim/automaton/state.py

```diff
@@ -1,61 +1,59 @@
+"""State class for a FSM. Each state should inherit from this class."""
+
 class State():
-    """
-    Our own state class, used by the Finite State Machine.
-    """
+    """Our own state class, used by the Finite State Machine."""
```

Comment:
Change comments to """Our own state class (acts as an 'interface'), used by the Finite State Machine."""